src/pages/Payment/SelectCard.js

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+
+import { useState } from "react";
```

Comment:
동일한 패키지에서 import하기 때문에 한 줄로 import하시면 될 것 같습니다.
```suggestion
import React, { useState } from "react";
```